platform/ui/src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Icon, Typography } from '@ohif/ui';
+
+const Dropdown = ({ children, showDropdownIcon, list }) => {
+  const [open, setOpen] = useState(false);
+  const element = useRef(null);
+
+  const DropdownItem = ({ title, icon, onClick, index }) => {
+    const itemsAmount = list.length;
+    const isLastItem = itemsAmount === index + 1;
```

Comment:
Maybe require int index if we're going to `+ 1`?

Reply:
Why do we need parent to provide index at all if iterator can apply key?

Reply:
removed this logic due to the tailwind class